lessons/Algebra2/logarithmic2-building-models/langs/en-us/index.adoc

```diff
@@ -53,56 +53,97 @@ You will want to:
 @objective{logarithms-exponents}
 
 === Overview
-Having identified that the wealth-v-lifespan scatter plot is neither linear, nor quadratic, nor exponential, students are asked to make connections to exponential functions.
+Having identified that the wealth-v-lifespan scatter plot is neither linear, quadratic, or exponential, students are asked to make connections to exponential functions.
 
 === Launch
 
 @lesson-instruction{
 - Open @starter-file{alg2-countries}, and make a scatter plot showing the relationship between `pc-gdp` and `median-lifespan`, using the name of the `country` as the label.
 }
 
-All of the models we just found for this dataset made predictions with too much error. Even though this shape might look _familiar_, it's just not any kind of growth we've seen before.
+All of the models we just found for this dataset made predictions with too much error. This shape just isn't any kind of growth we've seen before, although this shape might look _familiar_...
 
 @slidebreak
 
-Exponential growth started really _slowly_, and then suddenly shot up like a rocket to numbers so large we can't even reason about them. This kind of growth feels like the exact opposite, growing really _quickly_ but then leveling off to something very slow.
+@lesson-instruction{
+- Complete the first section of @printable-exercise{from-expt-to-log.adoc}
+- What do you notice about the first two scatter plots?
+}
+
+Exponential growth starts _slowly_, and then explodes to enormous numbers so quickly that we have trouble predicting what the next value will be. But our Life Expectancy v. Income growth feels like the exact opposite, growing really _quickly_ but then leveling off to something very slow.
 
 @slidebreak
 
-@right{@image{images/covid-graph.png, 375}}Let's think back to the graph of our model for Covid spread, from @lesson-link{exponential1-exploring-covid}.
+@right{@image{images/MA-covid.png, 275}}Consider the growth of Covid Infections from Massachusetts (from @lesson-link{exponential1-exploring-covid}).
 
-- Our model answers the question "how many positive cases do we predict after @math{x} days?"
+- The model for this data answers the question "how many positive cases do we predict after @math{x} days?"
 - The graph is showing us the relationship from +
 _days → cases_, _x → y_, or _Domain → Range_.
 
-But what if we want to answer the _opposite_ question: "how many days will it take to reach @math{y} cases?"
+The kind of growth we're seeing in the Countries dataset looks a lot like the Covid graph with the axes swapped!
 
 - Now we want to *flip the relationship around*, asking for +
 _cases → days_, _y → x_, or _Range → Domain_.
 - What would *this* graph look like?
 
 @slidebreak
 
-@ifslide{@right{@image{images/covid-graph.png, 350}}}
+++++
```

Comment:
I have questions about this animation. 

- Do we expect it to work on slides?
- Is it possible not to have it reverse, and have it go blank, reset, and start over instead?